### DISCUSSION TITLE
Help on Faithfulness Eval

### DISCUSSION THREAD

**Discussion by siddartha-10** [confident-ai/deepeval#1232] [Q&A]
I am trying to do faithfulness eval on my RAG application, I have about 6.5k rows of data. How do I run it faster, in parallel. Any help would greatly appreciated.

**Comment by kritinv** [comment]
Hey @siddartha-10, DeepEval supports async processing, parallelization, and caching to speed up your evaluations. You can configure parameters like `run_async`, `throttle_value`, and `max_concurrent` to control async behavior. Linking the docs for you here:

- Async: https://www.deepeval.com/docs/evaluation-test-cases#evaluate-test-cases-in-bulk
- Parallelization: https://www.deepeval.com/docs/evaluation-introduction#parallelization
